cabal.project.freeze

```diff
@@ -1,19 +1,26 @@
+-- NOTE: Due to revisions, this file may not work. See:
+-- https://github.com/fpco/stackage-server/issues/232
```

Comment:
This is straight copy-pasta from https://www.stackage.org/lts-13.16/cabal.config